webcompat/templates/home-page/form.html

```diff
@@ -57,7 +57,7 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
             </div>
             <span class="wc-Form-validation">
             {{ form.description(class_='wc-Form-item required',
-               placeholder='e.g. The video should have started after 3 sec.',
+               placeholder="e.g. The video doesn't play",
```

Comment:
@zoepage it appeared the convention was to use double quotes for strings now. cannot recall now whether this was an issue from the prettier tool or not. #1347